### PR TITLE
TensorflowLite unit test error fix

### PR DESCRIPTION
**Pull request by mgokulkrish**
Resolving errors which occur during `bazel test //tensorflow/lite/delegates/gpu/common/...`

This fixes the errors in unit_tests in:

fully_connected.
lstm.
mean_stddev_normalization.
interpreter_utils.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @mgokulkrish Can you please sign CLA. Thank you!

**Comment by mgokulkrish** [comment]
Hi @gbaned , i had signed a cla yesterday under the name mgokulkrish and email gokulkrishna98@gmail.com. Maybe i missed something, I am new to open source.

**Comment by mgokulkrish** [comment]
Hi @gbaned, I think the cla workflow has verified my id.

**Comment by gbaned** [comment]
> Hi @gbaned, I think the cla workflow has verified my id.

Hi @mgokulkrish  Yes, It is fine now. Thank you!

**Comment by mgokulkrish** [comment]
Hi @terryheo, could you please review this PR. Doing open-source contribution for the first time, would love to learn from the mistakes and improve.

**Comment by mgokulkrish** [comment]
Hi @gbaned, This PR is yet to be reviewed or approved, could you add more reviewers.

**Comment by gbaned** [comment]
Hi @terryheo / @sirakiin Can you please review this PR ? Thank you!

File: tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc [hunk 1]
```diff
@@ -50,12 +50,17 @@ absl::Status InterpreterInvokeWithOpResolver(
     return absl::InternalError("Unable to allocate TfLite tensors");
   }
   for (int i = 0; i < inputs.size(); ++i) {
-    DCHECK_EQ(interpreter->tensor(interpreter->inputs()[i])->type,
-              kTfLiteFloat32);
+    if(interpreter->tensor(interpreter->inputs()[i])->type !=  kTfLiteFloat32){
```
Comment (terryheo) [inline tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc:53]: Could you explain why this change is needed?

Comment (mgokulkrish) [reply tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc:53]: Hi @terryheo , When I ran the bazel test command, it gave error saying the DCHECK_EQ macro/function was not defined.

Comment (terryheo) [reply tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc:53]: Oh I see. We might consider using TFLITE_DCHECK_EQ but this looks fine to me. Thanks!

**Review by terryheo — APPROVED** [review]
LGTM

File: tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc [hunk 2]
```diff
@@ -67,14 +67,12 @@ absl::Status LstmTest(TestExecutionEnvironment* env) {
       RETURN_IF_ERROR(
           PointWiseNear({7.0 / 15.0, 10.0 / 15.0, 13.0 / 15.0, 16.0 / 15.0},
                         new_state.data, eps))
```
Comment (terryheo) [inline tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc:69]: You need to put ';' at the end of line.

File: tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc [hunk 3]
```diff
@@ -67,14 +67,12 @@ absl::Status LstmTest(TestExecutionEnvironment* env) {
...
-          << ToString(storage) << ", " << ToString(precision);
       RETURN_IF_ERROR(PointWiseNear(
           {static_cast<float>((1.0 / 6.0) * std::tanh(7.0 / 15.0)),
            static_cast<float>((1.0 / 6.0) * std::tanh(10.0 / 15.0)),
            static_cast<float>((1.0 / 6.0) * std::tanh(13.0 / 15.0)),
            static_cast<float>((1.0 / 6.0) * std::tanh(16.0 / 15.0))},
           new_activ.data, eps))
```
Comment (terryheo) [inline tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc:75]: ditto

File: tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc (hunk 2 above)
Comment (mgokulkrish) [reply tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc:69]: Hi @terryheo, I think the ; is not necessary, I am assuming this is macro from this path
tensorflow/lite/delegates/gpu/common/google/status_macros.h

`
#define RETURN_IF_ERROR(s) \
  {                        \
    auto c = (s);          \
    if (!c.ok()) return c; \
  }
`

If there is error in my understanding, please correct.

**Comment by gbaned** [comment]
Hi @mgokulkrish Can you please check @terryheo's [comments](https://github.com/tensorflow/tensorflow/pull/60749#discussion_r1275270526) and keep us posted ? Thank you!

**Comment by mgokulkrish** [comment]
> Hi @mgokulkrish Can you please check @terryheo's [comments](https://github.com/tensorflow/tensorflow/pull/60749#discussion_r1275270526) and keep us posted ? Thank you!

Hi @gbaned, I have replied to @terryheo 's post. After clarification will make the required changes.

**Comment by gbaned** [comment]
Hi @terryheo Any update on this PR? Please. Thank you!

File: tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc (hunk 1 above)
Comment (terryheo) [reply tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc:53]: Could you apply TF code style?

https://www.tensorflow.org/community/contribute/code_style

Comment (mgokulkrish) [reply tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc:53]: Sure, will update the formatting.

**Comment by gbaned** [comment]
Hi @terryheo Any update on this PR? Please. Thank you!

**Comment by terryheo** [comment]
Let me fix the code style.
